src/main/java/mil/dds/anet/config/AnetConfiguration.java

```diff
@@ -146,6 +171,69 @@ public void setDictionary(Map<String, Object> dictionary) {
     this.dictionary = Collections.unmodifiableMap(dictionary);
   }
 
+  @SuppressWarnings("unchecked")
+  public void loadDictionary() {
+    // Read and set anet-dictionary
+    final File file = new File(System.getProperty("user.dir") + "/anet-dictionary.yml");
```

Comment:
we need to pass the name of the dictionary file as an argument. Either on the command line or as part of anet.yml

Reply:
Now I've added a new parameter to anet.yml which holds the name of dictionary.While AnetConfiguration is prepared to set configuration it passes the name of the file as parameter to its "set" method and set it. Then right after "loadDictionary" method is called in that "set" method to load dictionary.
In this way we do not need to change any command line script calls and also do not need to update build gradle file as well.

For example "bin/anet check anet.yml" command remains the same way as it is .No need to add a new parameter.
I've tested some command line scenarios as below :
check from command line with invalid dictionary : throws expected exception and stops
check from command line with missing dictionary in anet folder : throws expected exception and stops

another scenario with already started Anet server with a valid dictionary: 
// try to reload with an invalid dictionary through admin UI
step 1 : start Anet with a valid dictionary 
step 2 : change dictionary with invalid data manually
step 3 : open admin UI and press reloadDictionary
result : it produces expected error message and does not set invalid dictionary as expected

// try to reload dictionary with new and valid values through admin UI
step 1 : start Anet with a valid dictionary 
step 2 : change dictionary manually with proper data 
step 5 : open admin UI and press reloadDictionary
result : it reloads dictionary with new values successfully